YAXLib/YAXSerializer.cs

```diff
@@ -2833,7 +2833,7 @@ private IEnumerable<MemberWrapper> GetFieldsToBeSerialized(UdtWrapper typeWrappe
         /// <returns>the sequence of fields to be serialized for the serializer's underlying type.</returns>
         private IEnumerable<MemberWrapper> GetFieldsToBeSerialized()
         {
-            return GetFieldsToBeSerialized(m_udtWrapper);
+            return GetFieldsToBeSerialized(m_udtWrapper).OrderBy(t=>t.Order);

```

Comment:
Yeah, I was thinking I'd write a test for this case. How do I append this pull request?